src/addremove.js

```diff
@@ -0,0 +1,63 @@
+export const toDoList = JSON.parse(localStorage.getItem('toDoList')) || [];
+
+let index = toDoList.length;
+
+export const generateElement = (index, completed, description) => {
+  const ulLists = document.querySelector('.ul-lists');
+  const list = document.createElement('li');
+  list.setAttribute('class', 'list');
+  list.setAttribute('id', `${index}`);
+  ulLists.appendChild(list);
+  const span = document.createElement('span');
+  span.setAttribute('class', 'list-check');
+  list.appendChild(span);
+
+  const checkbox = document.createElement('input');
+  checkbox.setAttribute('type', 'checkbox');
+  checkbox.checked = completed;
+  span.appendChild(checkbox);
+
+  const paragraph = document.createElement('p');
+  paragraph.textContent = description;
+  span.appendChild(paragraph);
+
+  const ellipsis = document.createElement('i');
+  ellipsis.setAttribute('class', 'fa fa-ellipsis-v');
+  list.appendChild(ellipsis);
+};
+
+export const addItem = (description, completed) => {
+  const item = { description, index, completed };
+  if (description !== '') {
+    index += 1;
+    item.index = index;
+    toDoList.push(item);
+    localStorage.setItem('toDoList', JSON.stringify(toDoList));
```

Comment:
- [ ] Good work implementing the `add` and `remove` task functionality. However, the `edit` task functionality is missing. Kindly implement it and ensure that the changes are saved in the local storage so that whenever the page is refreshed, data is retained

![image](https://user-images.githubusercontent.com/98453979/194497159-11e87b6d-1cfc-4414-846d-8b0a66cf7931.png)